database/process_query.py

```diff
@@ -0,0 +1,23 @@
+import os
+import csv
+import subprocess 
+
+# Creates new multiqc report given the absolute path for file_names.txt and the user specified directory
+def run_multiqc(file_names, user_dir, html = None):
```

Comment:
Just minor nit and suggestion to make it even more clear:

```suggestion
def run_multiqc(file_names, output_dir, open_html = False):
```

Reply:
yeah good point lol